sql/views.sql

```diff
@@ -403,5 +403,15 @@ CREATE OR REPLACE VIEW timescaledb_information.chunk_compression_settings AS
 		FROM unnest(s.orderby, s.orderby_desc, s.orderby_nullsfirst) un(orderby, "desc", nullsfirst)
 	) un ON true;
 
+
+CREATE OR REPLACE VIEW timescaledb_information.hypertable_columnstore_settings
+AS SELECT * FROM timescaledb_information.hypertable_compression_settings;
+
+CREATE OR REPLACE VIEW timescaledb_information.chunk_columnstore_settings AS
+SELECT * FROM timescaledb_information.chunk_compression_settings;
+
+CREATE OR REPLACE VIEW timescaledb_information.columnstore_settings AS
+SELECT * FROM timescaledb_information.compression_settings;
```

Comment:
I've never understood why we have three similar views. Why not have one for settings and then you can see in  the view if it is a chunk or hypertable.

We should take the opportunity to reconsider the structure here if we are introducing new names.

Reply:
Let's not just replicate these views like this.

Unless I am mistaken, `compression_settings` is a legacy view kept for backwards compatibility so we should not move it over to the new naming format. I think we can merge chunks and hypertables in a single `columnstore_settings` view. There's no reason to split the same information across two views as one usually filters on relname or type when querying anyway.